server/mdm/apple/commander.go

```diff
@@ -118,7 +118,14 @@ func (svc *MDMAppleCommander) DeviceLock(ctx context.Context, host *fleet.Host,
 	return nil
 }
 
-func (svc *MDMAppleCommander) EraseDevice(ctx context.Context, hostUUIDs []string, uuid string) error {
+func (svc *MDMAppleCommander) EraseDevice(ctx context.Context, host *fleet.Host, uuid string) error {
+	// TODO(mna): do we need this PIN, and if so should it be saved somewhere? It
+	// doesn't appear in the Wipe host's spec/figma like we need one and it
+	// doesn't look like it is required by Apple:
+	// https://developer.apple.com/documentation/devicemanagement/erasedevicecommand/command.
+	// Although it does mention that this is for "Find my device", so maybe we
+	// don't have to store it but if the user is an admin on Apple's website they
+	// can view the PIN somewhere in there so it's still useful?
```

Comment:
Open question. I don't think we need to store it explicitly, we don't need/use it anywhere in the UI/CLI and worse case I think we can always extract it from the raw MDM command that will be stored.

Reply:
I don't think we need to store it either, given the current spec for this feature. @roperzh any thoughts?

Reply:
Thanks y'all, I'll remove the TODO and keep things as-is (wipe PIN not stored, but retrievable via the XML if needed in the future).